src/ui/server/helpers/map-eligibility-answers/index.ts

```diff
@@ -16,7 +16,8 @@ const {
  */
 const mapEligibilityAnswers = (answers: SubmittedDataInsuranceEligibility) => {
   if (answers.buyerCountry) {
-    const { buyerCountry, totalContractValue, coverPeriod, hasReviewedEligibility, ...otherAnswers } = answers;
+    // TODO: EMS-3680 3682 remove partyToConsortium, memberOfAGroup
```

Comment:
Is this temporarily for DB updates/updating application creation?

Reply:
yes, will be removed in a couple of PRs time